statusboard/src/utils/utils.ts

```diff
@@ -112,15 +112,26 @@ export function filterProjectsByCfA(
   nonCfA?: string
 ) {
   if (nonCfA === 'true') {
-    console.log("returning all projects");
+    
     console.log(projects.length);
     return projects;
   }
-  console.log("filter only cfa projects");
+  
   return projects.filter((p: Project) =>
     p?.brigade?.type ? p.brigade.type.includes("Brigade") || p.brigade.type.includes("Code for America") : false
   );
+}
 
+export function filterProjectsByText(
+  projects: Project[],
+  text?: string
+){
+if (!text){
+return projects;
+}
+return projects.filter((p:Project)=>
+p?.description?.includes(text) || p?.brigade?.type?.includes(text) || p?.name?.includes(text)
+)
```

Comment:
Perhaps shelve the changes that are not going to be used in this PR, so you have easy access to them in a future PR?

Unfortunately, I don't know how VSCode deals with shelving/stashing commits, so I can't really help with the practical aspect of it :/

Reply:
You are correct of course. It wasn't just this file however so I've gone through and removed all the references to global filtering, not just this function. I did stash all changes as well.